x-pack/filebeat/filebeat.reference.yml

```diff
@@ -1878,136 +1878,134 @@ filebeat.modules:
     #var.password:
 
 #------------------------------ Salesforce Module ------------------------------
-- module: salesforce
-
-  apex-rest:
-    enabled: false
-
-    # Oauth Client ID
-    #var.client_id: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
-
-    # Oauth Client Secret
-    #var.client_secret: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
-
-    # Oauth Token URL
-    #var.token_url: "https://login.salesforce.com/services/oauth2/token"
-
-    # Oauth User, should include the User mail
-    #var.user: "abc.xyz@mail.com"
-
-    # Oauth password, should include the User password
-    #var.password: "P@$$W0₹D"
-
-    # URL, should include the instance_url
-    #var.url: "https://instance_id.my.salesforce.com"
-      
-  login-rest:
-    enabled: false
+# Configuration file for Salesforce module in Filebeat
 
-    # Oauth Client ID
-    #var.client_id: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+# Common Configurations:
+# - enabled: Set to true to enable ingestion of Salesforce module fileset
+# - initial_interval: Initial interval for log collection. This setting determines the time period for which the logs will be initially collected when the ingestion process starts, i.e. 1d/h/m/s
+# - log_file_interval: Interval type for log file collection, either Hourly or Daily

```

Comment:
Doesn't this just apply to ELF API? Hourly/ daily? Should this be a common config?